### PR TITLE
Adding Project - Tinyxml2

### PR DESCRIPTION
**Pull request by jumde**

### CODE REVIEW

**Comment by jumde** [comment]
I signed it!

**Comment by jumde** [comment]
I signed it!

**Comment by jumde** [comment]
I signed it!

**Comment by inferno-chromium** [comment]
Please move fuzz target and dictionary in your repository, this helps with maintenance and better regression testing, see https://github.com/google/oss-fuzz/blob/master/docs/ideal_integration.md

File: projects/tinyxml2/build.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+make -j$(nproc) clean
+make CXXFLAGS='' -j$(nproc) all
```
Comment (Dor1s) [inline projects/tinyxml2/build.sh]: Resetting `CXXFLAGS` is wrong, as it disables both ASan instrumentation and coverage as well.

Comment (Dor1s) [reply projects/tinyxml2/build.sh]: Removing `CXXFLAGS=''` seems to fix the issue without any breakages, but I'd like to wait a little to make sure that https://github.com/google/oss-fuzz/pull/814 would catch a case like this one. After that I'll land the fix.

Comment (Dor1s) [reply projects/tinyxml2/build.sh]: The issue has been detected in expected way, uploading the fix for `tinyxml2` now.

```
Step #4: ++ sancov -print-coverage-pcs /workspace/out/address/xmltest
Step #4: ++ wc -l
Step #4: BAD BUILD: the code does not seem to have coverage instrumentation.
Step #4: + local NUMBER_OF_EDGES=16
Step #4: + ((  16 < 90  ))
Step #4: + echo 'BAD BUILD: the code does not seem to have coverage instrumentation.'
Step #4: + CHECK_FAILED=1
Step #4: + ((  1 > 0  ))
Step #4: + exit 1
```

Comment (Dor1s) [reply projects/tinyxml2/build.sh]: Done in https://github.com/google/oss-fuzz/commit/20fc8716df00bd272d0e26d380bc1bca0dad38d7

**Pull request merged**